packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```diff
@@ -0,0 +1,366 @@
+{
+    "About Us": "À propos de nous",
+    "Account": "Compte",
+    "accountChip.chipText": "Salut, {name}",
+    "accountInformationPage.accountInformation": "Information sur le compte",
+    "accountInformationPage.editAccount": "Modifier les informations du compte",
+    "accountInformationPage.errorTryAgain": "Un problème est survenu. Veuillez actualiser et réessayer.",
+    "accountInformationPage.titleAccount": "Information sur le compte - {name}",
+    "accountMenu.accountInfoLink": "Information sur le compte",
+    "accountMenu.addressBookLink": "Carnet d'adresses",
+    "accountMenu.communicationsLink": "Les communications",
+    "accountMenu.favoritesListsLink": "Listes de favoris",
+    "accountMenu.orderHistoryLink": "Historique des commandes",
+    "accountMenu.savedPaymentsLink": "Paiements enregistrés",
+    "accountMenu.signOutButtonText": "Déconnexion",
+    "accountMenu.storeCreditLink": "Cartes de crédit et cadeaux en magasin",
+    "accountTrigger.ariaLabel": "Toggle Menu Mon compte",
+    "accountTrigger.buttonFallback": "Se connecter",
+    "accountTrigger.signIn": "Se connecter",
+    "addressBook.addNewAddresstext": "Ajouter une nouvelle adresse",
+    "addressBook.applyButtonText": "Appliquer",
+    "addressBook.cancelButtonText": "Annuler",
+    "addressBook.headerText": "Modifier les informations de livraison",
+    "addressBookPage.addAddressText": "Ajouter une adresse",
+    "addressBookPage.addressBookText": "Carnet d'adresses",
+    "addressCard.defaultText": "Défaut",
+    "app.errorOffline": "Tu es hors ligne. Certaines fonctionnalités peuvent ne pas être disponibles.",
```

Comment:
Second-person singular (**Tu es**) is used in a familiar way in french, eg : when speaking to a friend of family. I don't know if this was the intend here. Using second-person plural (**Vous êtes**) is usually the standard in french when we want to be polite / be professional / speaking to a stranger.